src/main/java/ch/kalunight/zoe/repositories/LeaderboardRepository.java

```diff
@@ -0,0 +1,181 @@
+package ch.kalunight.zoe.repositories;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import ch.kalunight.zoe.model.dto.DTO;
+
+public class LeaderboardRepository {
```

Comment:
Codacy found an issue: [The utility class name 'LeaderboardRepository' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=5821508)